workflows/flows/assemble_study_tasks/assemble_samplesheets.py

```diff
@@ -28,6 +29,72 @@
 EMG_CONFIG = settings.EMG_CONFIG
 
 
+HOST_TAXON_TO_REFERENCE_GENOME = {
+    "9031": "chicken.fna",  # Gallus gallus
+    "8030": "salmon.fna",  # Salmo salar
+    "8049": "cod.fna",  #  Gadus morhua
+    "9823": "pig.fna",  # Sus scrofa
+    "9913": "cow.fna",  # Bos taurus
+    "10090": "mouse.fna",  # Mus musculus
+    "7460": "honeybee.fna",  # Apis mellifera
+    "10116": "rat.fna",  # Rattus norvegicus
+    "8022": "rainbow_trout.fna",  # Oncorhynchus mykiss
+    "9940": "sheep.fna",  # Ovis aries
+    "3847": "soybean.fna",  # Glycine max
+    "7955": "zebrafish.fna",  #  Danio rerio
```

Comment:
We need to re-oganize these files :(. I was aiming of having something like this https://ewels.github.io/AWS-iGenomes/ but in our FTP|hps.

Reply:
Cool, intriguing. Is that something near-term @mberacochea (i.e. should I keep this PR open and wait for it) or one to refactor later?

Reply:
I don't think so... I was leaving those thoughts here for posterity. There is a JIRA task, but we won't be working on this in the short term.